packages/backend-openapi-utils/api-report.md

```diff
@@ -248,6 +248,9 @@ type FullMap<
   },
 > = RequiredMap<T> & OptionalMap<T>;
 
+// @public
+export function getOpenApiSpecRoute(baseUrl: string): string;
```

Comment:
What do you think about only making this one public API? Skipping `OPENAPI_SPEC_ROUTE`